### PR TITLE
Adding new Inovelli vzm31-sn parameters

### PR DESCRIPTION
**Pull request by michaelkkehoe**
vzm31-sn firmware 2.14 added/changed parameters on these switches. This change is to reflect those.

### CODE REVIEW

**Review by epower53 — APPROVED** [review]
parameter IDs and data types match what was specified by the manufacturer

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -4948,10 +4948,14 @@ const Cluster: {
             activeEnergyReports: {ID: 0x0014, type: DataType.uint16},
             powerType: {ID: 0x0015, type: DataType.boolean},
             switchType: {ID: 0x0016, type: DataType.uint8},
+            higherOutputInNonNeutral: {ID: 0x0019, type: DataType.boolean},
             buttonDelay: {ID: 0x0032, type: DataType.uint8},
             deviceBindNumber: {ID: 0x0033, type: DataType.uint8},
             smartBulbMode: {ID: 0x0034, type: DataType.boolean},
-            doubleTapUpForFullBrightness: {ID: 0x0035, type: DataType.boolean},
+            doubleTapUpToParam55: {ID: 0x0035, type: DataType.boolean},
+            doubleTapUpToParam56: {ID: 0x0036, type: DataType.boolean},
```
Comment (corporategoth) [inline src/zcl/definition/cluster.ts:4956]: Down?

Comment (epower53) [reply src/zcl/definition/cluster.ts:4956]: good catch! should be down for this one

**Comment by Koenkk** [comment]
Merged https://github.com/Koenkk/zigbee-herdsman/pull/694 instead (as it was created earlier)

**Pull request closed without merging**